auth.go

```diff
@@ -134,12 +141,7 @@ func (m *MenderAuthManager) MakeAuthRequest() (*AuthRequest, error) {
 		return nil, errors.Wrapf(err, "failed to obtain device public key")
 	}
 
-	tentok, err := m.store.ReadAll(authTenantTokenName)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to read tenant token")
-	}
-
-	tentok = bytes.TrimSpace(tentok)
+	tentok := bytes.TrimSpace([]byte(m.tenantToken))
```

Comment:
Yeah, totally forgot about `strings.TrimSpace()`.

As for validating the token, we could check if it's not empty (like it was before). Minor issue though, the backend does not care about the token, and frankly I don't know if it's planned to keep it like this or not in the future. IMO, once the backend starts caring, there should be a version bump in the API and we should check that in the Auth client code.